doc/nrf/ug_multi_image.rst

```diff
@@ -152,6 +153,37 @@ See the following example:
 
 A *domain* is well-defined if there is the ``CONFIG_DOMAIN_${DOMAIN}_BOARD`` configuration option in Kconfig.
 
+.. _ug_multi_image_add_child_image_oot:
+
+Adding a child image using Zephyr modules
+=========================================
+
+Any call to ``add_child_image`` must be done *after* :file:`nrf/cmake/extensions.cmake` is invoked, but *before* :file:`multi_image.cmake` is invoked.
+In some scenarios, this is not possible without modifying the NCS build code, for example, from top-level sample files and project :file:`CMakeLists.txt` files.
+
+To avoid this issue, use the *Modules* mechanism provided by the Zephyr build system.
+The following example shows how to add the required module from a top-level sample :file:`CMakeLists.txt`.
+
+.. literalinclude:: ../../samples/nrf5340/multicore/CMakeLists.txt
+    :language: cmake
+    :start-at: cmake_minimum_requirement
```

Comment:
```suggestion
    :start-at: cmake_minimum_requirement
```

@hakonfam I think there is a typo, either here or in the cmakelists.txt file (that mentions "cmake_minimum_required")

Reply:
fixed